NocWorx/ruleset.xml

```diff
@@ -27,6 +27,9 @@
     <exclude name="Squiz.WhiteSpace.ScopeClosingBrace.ContentBefore"/>
     <exclude name="Squiz.ControlStructures.ForLoopDeclaration"/>
     
+    <!-- We have at least one case (Invoices.php:237) where the length of the line exceeds 80 chrs. The only solution is to break the line at the ()'s evne though there are no parameters. We can fix this by adding an "ignoreNewLines" parameter but I haven't yet. Once that is implemented, remove this exclude. -->
```

Comment:
is this referring to something in core? could you reply with a link to the line of code so i can have a look?

Reply:
Yes. It's more of a note for me than anything else. Let me put the full core path with the file name.

Reply:
I've removed this exclude. Right now there are no files breaking this rule. The problem will arise when we start putting long class names as hints. Specifically Return Type hints. Once we start putting typehints in and trigger this, I'll remove it.